frontend/src/components/Smart/index.jsx

```diff
@@ -35,7 +35,7 @@ class Smart extends React.Component {
             adminTabSkew = (
                 <Tab eventKey={3} transition={false} title="Fix Skew" className="full card">
                     <div className="cardContent">
-                        <h2>Another admin is currently using this page. Please check back later.</h2>
+                        <h2>Another admin is currently using this page. This page will become available when the admin returns to the project list page, changes projects, or logs out.</h2>
```

Comment:
Agree with Durk's comment. Would also include something like "this page will become available **upon refresh** ..." or some similar instruction to refresh. I am imagining a scenario where someone gets this error message, slacks the team to have admins change projects, and then doesn't see anything update (because they need to manually refresh the page)

Could consider changing "Another admin" to a username if it's straightforward, but I'm not particularly attached to this feature unless someone requests it

Reply:
honestly it's a decent amount of work to add it, so maybe we could put it as an issue to get this merged in?

Reply:
also updated admin locked message to include details page